travel_platform/travel_platform/settings.py

```diff
@@ -131,3 +131,30 @@
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+LOGGING = {
```

Comment:
Did you have a chance to google if it's a common way to have a logging configurations in Django config?
Because I'm not sure about it.

In non-Django projects, we always had a separate file for logging configuration like `logging.conf`, etc.

I'm OK if this approach is common in `Django world`.